src/cloudinary/helper.js

```diff
@@ -0,0 +1,93 @@
+import { A11Y_TRANSFORMS, progressive, PLACEHOLDER_OPTIONS, COMPONENTS, predominantColorTransformPxl } from '../constants';
+
+/**
+ * 
+ * @param {Boolean} enable 
+ * @returns {Object} transformation object for progressive effect
+ */
+const addProgressive = (enable = false) => {
+  return enable ? progressive : {}
+}
+
+/**
+ * 
+ * @param {'darkmode'|'brightmode'|'monochrome'|'colorblind'} type 
+ * @returns {Object} transformation effect for a11y support based on type
+ */
+const addA11y = (type) => {
+  return A11Y_TRANSFORMS[type] || {}
+}
+
+/**
+ * 
+ * @param {Object} object contains accessibility, withProgressive, cldTransforms and baseOptions
+ * @returns {Object} options for generating delivery URL of a media component 
+ */
+export const computeOptions = ({ accessibility, withProgressive, baseOptions = {}, extra = [] }) => {
+  const transformation = [...extra]
+  const a11y = addA11y(accessibility)
```

Comment:
Personally, for me, a11y is less readable than accessibility.
I know it's shorter to write, but that's just my preference.

It doesn't help that 1 looks like an l, and I'm reading it at as 'ally' all the time.

Reply:
well, a11y is well-known worldwide for accessibility :). But we can always go for the longer version :)